homeassistant/components/azure_data_explorer/client.py

```diff
@@ -72,6 +72,11 @@ def __init__(self, data: Mapping[str, Any]) -> None:
 
         self.query_client = KustoClient(kcsb_query)
 
+        # Suppress very verbose logging from client
+        logger = logging.getLogger("azure")
+        logger.setLevel(logging.WARNING)
```

Comment:
Can't we change this in the library?

Reply:
According to this: https://learn.microsoft.com/en-us/azure/developer/python/sdk/azure-sdk-logging this is they way. however Im now a bit more precise, using azure.kusto